includes/class-subscription-lists.php

```diff
@@ -0,0 +1,121 @@
+<?php
+/**
+ * Newspack Subscription Lists
+ *
+ * @package Newspack
+ */
+
+namespace Newspack\Newsletter;
```

Comment:
```suggestion
namespace Newspack\Newsletters;
```

The plugin is called "Newsletters"